components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -303,6 +303,28 @@ Chef Infra Server lets you delete the existing clients one at a time. To delete
 
 {{< figure src="/images/automate/delete-a-client.png" alt="Delete a Client">}}
 
+### Nodes
+
+A [Node]({{< relref "nodes" >}}) is a communication between the Infra Chef Server and the Infra Clients. Nodes are used when an Infra Client asks for a policy from the Infra Chef Server.
+
+The Chef Infra Server UI lets you:
+
+- Search for a specific Node from a list of nodes.
+- List out all the nodes at one place.
+- Delete a node
+
+#### Search for a Node
+
+Use the **Search nodes by name** bar to find a node from the list of nodes. Entering the name of a node in the search bar returns nodes matching your search criteria.
```

Comment:
```suggestion
Use the search bar on the Nodes tab (**Chef Infra Servers > Nodes**)  to find a node from the list of nodes. Entering the name of a node in the search bar returns nodes matching your search criteria.
```

Reply:
I haven't tried it, but is there anything hinky (not quite right) about how search works? Also, does it support wildcards?